.gitignore

```diff
@@ -1,34 +1,11 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
-# compiled output
-/dist/
-/tmp/
-
 # dependencies
-/bower_components/
-/node_modules/
+node_modules/
 
 # misc
-/.env*
```

Comment:
I think pretty much everything should stay as it was in this file and the old contents should be copied to the new ember-flatpickr folder, I think, to ensure the right things are ignored. Especially stuff like ember-try, which we want to keep.

Reply:
Just to be sure, you prefer to keep the original content of this file at the project root ?